c_examples/libmicrovmi.h

```diff
@@ -0,0 +1,82 @@
+#ifndef LIBMICROVMI_H
```

Comment:
We shouldn't include a generated file in the sources.
It will be outdated in a few commits, and the user should generate this file by himself.